.NET/Microsoft.Recognizers.Text.DateTime/Chinese/Parsers/ChineseDatePeriodParserConfiguration.cs

```diff
@@ -964,27 +975,15 @@ private DateTimeResolutionResult MergeTwoTimePoints(string text, DateObject refe
                 pastEnd = futureEnd;
             }
 
+            ret.Timex = TimexUtility.GenerateDatePeriodTimex(futureBegin, futureEnd, DatePeriodTimexType.ByDay, pr1.TimexStr, pr2.TimexStr);
+
             if (pr1.TimexStr.StartsWith(Constants.TimexFuzzyYear) && futureBegin.CompareTo(DateObject.MinValue.SafeCreateFromValue(futureBegin.Year, 2, 28)) <= 0 && futureEnd.CompareTo(DateObject.MinValue.SafeCreateFromValue(futureBegin.Year, 3, 1)) >= 0)
             {
                 // due to only solve the noYear cases, we only have to judge the period in one year contains Feb 29th;
-                ret.Comment = Constants.Comment_ContainFeb29th;
-                ret.FutureResolution = new Dictionary<string, string>
-                {
-                    {
-                        DateTimeResolutionKey.Timex,
-                        $"({pr1.TimexStr},{pr2.TimexStr},P{(futureEnd - futureBegin).TotalDays}D)"
-                    },
-                };
-                ret.PastResolution = new Dictionary<string, string>
-                {
-                    {
-                        DateTimeResolutionKey.Timex,
-                        $"({pr1.TimexStr},{pr2.TimexStr},P{(pastEnd - pastBegin).TotalDays}D)"
-                    },
-                };
+                ret.Comment = Constants.Comment_DoubleTimex;
+                ret.Timex += "|" + TimexUtility.GenerateDatePeriodTimex(pastBegin, pastEnd, DatePeriodTimexType.ByDay, pr1.TimexStr, pr2.TimexStr);

```

Comment:
It's better to be explicit on what's happening here. Create a new method called MergeTimeAlternatives in TimexUtility and pass both timexes to it.
And if you want to use a separator like '|' declare it as a constant.